lib/salestation/web/ip_address.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Salestation
+  class Web < Module
+    module IPAddress
+
+        def self.extract(request)
```

Comment:
this project doesn't have rubocop, so you'll have to fix the indentation yourself. The indentation should be two spaces. so this line should go left, and also there's indentation wrong inside the `if/else`.

Reply:
same issues in the tests

Reply:
isn't it fixed here? https://github.com/salemove/salestation/pull/67/commits/4b81c35ee264c111f0a7767069929eb2ceaa8d76
it show okay for me